subsys/bluetooth/controller/ll_sw/ull_adv_sync.c

```diff
@@ -507,6 +521,151 @@ void ull_adv_sync_offset_get(struct ll_adv_set *adv)
 	LL_ASSERT(!ret);
 }
 
+uint8_t ull_adv_sync_acad_enable(struct lll_adv_sync *lll_sync,
+				 uint8_t acad_len, void **acad)
+{
+	struct pdu_adv_com_ext_adv *ter_com_hdr, *ter_com_hdr_prev;
+	struct pdu_adv_ext_hdr *ter_hdr, ter_hdr_prev;
+	struct pdu_adv *ter_pdu, *ter_pdu_prev;
```

Comment:
Declaring multiple variables in a single line is usually less readable, but my comment is regarding the types her:

`struct pdu_adv_ext_hdr *ter_hdr, ter_hdr_prev;` declares both as pointers, right? 
Does `struct pdu_adv *ter_pdu, *ter_pdu_prev;` then not declare `ter_pdu_prev` as a `struct pdu_adv **`?